samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties

```diff
@@ -362,6 +362,13 @@ seb_config_mode = Nola konfiguratu nahi duzu Safe Exam Browser?
 seb_config_mode_client = Bezeroaren konfigurazioa erabili
 seb_config_mode_manual = Eskuz konfiguratu
 seb_config_mode_upload = Nire konfigurazioa igo
+seb_config_mode_client_help=Using client configuration will allow the use of encrypted or no SEB configuration. In this case no \
```

Comment:
Hi @PabloSanRoman, could you add this translations please?

Reply:
Hi @mpellicer 

+ seb_config_mode_client_help= Bezeroaren konfigurazioa erabiltzeak aukera emango du konfigurazio zifratua erabiltzeko edo SEB gabekoa erabiltzeko. Kasu honetan, Sakaik ez dio konfigurazio fitxategirik emango ikasleari.

Reply:
Thank you! Included in the S2U-21 work.